src/views/Clients.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <v-icon @click="add" v-if="isActive" large color="green">add_circle</v-icon>
+    <v-form @submit="submit" v-else method="post">
+      <v-text-field v-model="client.firstName" :counter="20" label="First Name"></v-text-field>
+      <v-text-field v-model="client.secondName" :counter="20" label="Second Name"></v-text-field>
+      <v-text-field v-model="client.phone" :counter="10" label="Phone Number"></v-text-field>
+      <v-btn
+        @click="listClients"
+        type="submit"
+        class="white--text success"
+        color="light-green accent-3"
+      >Submit</v-btn>
+      <v-btn class="white--text" color="red accent-2" @click="cancel">Cancel</v-btn>
+    </v-form>
+    <v-card>
+      <v-list>
+        <v-list-tile v-for="c in clients" :key="c.phone">
+          <v-list-tile-content>{{c.firstName}}</v-list-tile-content>
+          <v-list-tile-content>{{c.secondName}}</v-list-tile-content>
+          <v-list-tile-content>{{c.phone}}</v-list-tile-content>
+        </v-list-tile>
+      </v-list>
+    </v-card>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+const baseUrl = "http://localhost:3000/clients";
+
+export default {
+  data() {
+    return {
+      isActive: true,
+      client: {
+        firstName: "",
+        secondName: "",
+        phone: ""
+      },
+      clients: []
+    };
+  },
+  mounted() {
+    this.listClients();
+  },
+  methods: {
+    submit() {
+      axios.post(baseUrl, { body: this.client }).then(() => {});
+      this.isActive = true;
+      this.listClients();
+    },
+    add() {
+      this.isActive = false;
+    },
+    cancel() {
+      this.isActive = true;
+    },
+    listClients() {
```

Comment:
Please rename to fetchClients

Reply:
You see I commented line 59. So it was needed to rename `fetchClients` not `submit`